src/pilot/domain/answerValues.ts

```diff
@@ -19,6 +19,6 @@ export const FREMTIDIG_SITUASJON_ANSWER_TEXTS = {
 export const BEHOV_FOR_OPPFOLGING_ANSWER_TYPES = ['JA', 'NEI'] as const
 export type BehovForOppfolgingAnswerTypes = (typeof BEHOV_FOR_OPPFOLGING_ANSWER_TYPES)[number]
 export const BEHOV_FOR_OPPFOLGING_ANSWER_TEXTS = {
-  JA: 'Ja, jeg vil bli kontaktet',
-  NEI: 'Nei, jeg klarer meg selv',
+  JA: 'Ja, jeg vil bli kontaktet av en veileder',
```

Comment:
Kanskje vurdere å gå for enten inline-tekst eller som konstanter.

Vi har vel stort sett inline-tekster de fleste steder, men det går jo an å endre på om vi ønsker det.

Reply:
Vi har trukket det inn et objekt for å kunne gjenbruke verdiene for å kunne mappe tekst **og** bygge objekter med kun én nøkkel for å hente ut tilhørende verdi.

Men enig med at hvis det kun hadde vært for visning av tekst så kunne vi inlinet.